### PR TITLE
Fixes InfiniteLoader bug where stopRange is inclusive and rowsCount s…

### PR DESCRIPTION
**Pull request by khankuan**
…hould decrease by 1 when calling scanForUnloadedRanges

### CODE REVIEW

File: source/InfiniteLoader/InfiniteLoader.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ export default class InfiniteLoader extends Component {
     const unloadedRanges = scanForUnloadedRanges({
       isRowLoaded,
       startIndex: Math.max(0, startIndex - threshold),
-      stopIndex: Math.min(rowsCount, stopIndex + threshold)
+      stopIndex: Math.min(rowsCount - 1, stopIndex + threshold)
```
Comment (bvaughn) [inline source/InfiniteLoader/InfiniteLoader.js]: heh, whoops! :)

**Comment by bvaughn** [comment]
Thanks @khankuan :)

**Comment by khankuan** [comment]
You guys must have large data sets :)

**Comment by bvaughn** [comment]
This was just published as 6.2.2 :)

**Comment by bvaughn** [comment]
Haha, yeah, we do. But also I don't use `InfiniteLoader` but in one place I think. So I just don't pound on it much. I mostly wrote it for others to use. :)

**Pull request merged**